src/hyp3_isce2/dem.py

```diff
@@ -58,16 +59,39 @@ def buffer_extent(extent: list, buffer: float) -> list:
     ]
 
 
+def utm_from_lon_lat(lon: float, lat: float) -> int:
```

Comment:
Not blocking, but it might be good for utm_from_lon_lat and get_dem_resolution to have comments.